api/dotnet/src/Indigo.Net.csproj

```diff
@@ -97,10 +97,18 @@
 			<PackagePath>runtimes/win-x64/native/bingo-nosql.dll</PackagePath>
 			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
 		</Content>
+		<Content Include="../build/net461/Indigo.Net.targets">
+			<Pack>true</Pack>
+			<PackagePath>build/net461/</PackagePath>
+		</Content>
+		<Content Include="../buildTransitive/net461/Indigo.Net.targets">
+			<Pack>true</Pack>
+			<PackagePath>buildTransitive/net461/</PackagePath>
+		</Content>
 	</ItemGroup>
 
 	<Target BeforeTargets="PreBuildEvent" Name="PreBuild">
-		<Exec Command="powershell -ExecutionPolicy Bypass -NoProfile -NonInteractive -File $(ProjectDir)/prebuild.ps1" />
+		<Exec Command="powershell -ExecutionPolicy Bypass -NoProfile -NonInteractive -File $(ProjectDir)prebuild.ps1" />
```

Comment:
$(ProjectDir): The directory of the project (defined as drive + path); includes the trailing backslash '\'.